effekt/jvm/src/test/scala/effekt/LSPTests.scala

```diff
@@ -254,15 +254,17 @@ class LSPTests extends FunSuite {
     }
   }
 
-  // FIXME: Hovering over holes does not work at the moment.
-  // https://github.com/effekt-lang/effekt/issues/549
-  test("Hovering over hole shows nothing") {
+  test("Hovering over hole shows inside and outside types") {
     withClientAndServer { (client, server) =>
       val (textDoc, cursor) = raw"""
-                                |def foo(x: Int) = <>
-                                |                  ↑
+                                |def foo(x: Int): Bool = <{ x }>
```

Comment:
Would it also be possible to showcase the effect part? IIRC hovering over a hole ought to also show the effects inside and outside (which is super helpful for Effekt in particular :))